bin/tfidf_search.py

```diff
@@ -0,0 +1,97 @@
+import argparse
```

Comment:
Let's rename this to `zksim` (`zettelkasten-similar`) or `zkadj` (`zettelkasten-adjacent`) or something of that nature, to make it more consistent. You can then also add it to the README. FWIW, I'm fine with multiple languages (Ruby, Python, ..)